### ISSUE TITLE
Invalid REV dates should be ignored

### ISSUE THREAD

**Issue by devvv4ever** [bitfireAT/vcard4android#51]
I got a ticket with this problem here: 

Ticket: https://bitfire-at.zammad.com/#ticket/zoom/5137

```
BEGIN:VCARD
VERSION:3.0
UID:redacted
LANGUAGE:de
PRODID;VALUE=TEXT:-//Intermesh//NONSGML Group-Office 6.5.109//EN
N:redacted;redacted;;;
FN:redacted
REV;VALUE=DATE:15668479
TITLE:
EMAIL;TYPE=redacted
TEL;TYPE=work;VALUE=UNKNOWN:redacted
TEL;TYPE=mobile;VALUE=UNKNOWN:redacted
ADR;TYPE=home:;redacted
ORG:Telesis
NOTE:
X-GO-GENDER:redacted
CATEGORIES:redacted
END:VCARD
```

The value of 15668479 would be Year 1566, Month 84 Day 79 - which is not valid according to the standard.

Suggestion: If there are wrongly formatted REV dates occurring no error should be displayed since we don't need this value anywhere. We could safely ignore this, I think @rfc2822 ?.

Correct values in the rfc: https://datatracker.ietf.org/doc/html/rfc2426#section-3.6.4

**Comment by rfc2822** [comment]
Moved to vcard4android because this is about a vCard, not an iCalendar.

**Comment by rfc2822** [comment]
Duplicate of https://github.com/mangstadt/ez-vcard/issues/134, should be already fixed with 0.12.1 and thus latest vcard4android / DAVx⁵.
